models.py

```diff
@@ -163,22 +176,39 @@ def __str__(self):
             active=self.receives_notifications,
         )
 
+    def status_string(self):
```

Comment:
This looks to be a duplicate of the text in `type_of_notification_choices`. Inevitably we will eventually make a change to one and not the other...

Maybe we can make it a dictionary (`TYPE_OF_NOTIFICATION_CHOICES`) and have this method lookup the dict? Otherwise it might be possible to do self.model.type_of_notification.choices

Another option is to use `BillingStaffer._meta.get_field("type_of_notification").choices` but that is a tuple and requires accessing a private member, so having a top level dictionary might be best.